ml/src/main/scala/ai/h2o/sparkling/ml/algos/H2OGridSearch.scala

```diff
@@ -442,22 +441,18 @@ trait H2OGridSearchParams extends H2OCommonSupervisedParams {
 
   def getSelectBestModelBy(): String = $(selectBestModelBy)
 
-  def getSelectBestModelDecreasing(): Boolean = $(selectBestModelDecreasing)
+  @DeprecatedMethod("getSelectBestModelOrdering")
+  def getSelectBestModelDecreasing(): Boolean = {
+    val ordering = getSelectBestModelOrdering()
+    ordering == "ASC" // The default value was DESC as well
```

Comment:
`ordering == "DESC"` ?

What about the case  with `"AUTO"`?

Reply:
This won't be needed at the end as the option will just get deprecated without replacement in a form of `selectBestModelOrdering`